tutormfe/plugin.py

```diff
@@ -13,37 +13,51 @@
         "HOST": "apps.{{ LMS_HOST }}",
         "COMMON_VERSION": "{{ OPENEDX_COMMON_VERSION }}",
         "CADDY_DOCKER_IMAGE": "{{ DOCKER_IMAGE_CADDY }}",
+        "REMOVE_WEBPACK_BUILD_CACHE": False,
         "ACCOUNT_MFE_APP": {
             "name": "account",
-            "repository": "https://github.com/edx/frontend-app-account",
+            "repository": "https://github.com/openedx/frontend-app-account",
             "port": 1997,
-            "env": {
-                "production": {
-                    "COACHING_ENABLED": "",
-                    "ENABLE_DEMOGRAPHICS_COLLECTION": "",
-                },
-            },
+            "version":"open-release/olive.master",
         },
         "GRADEBOOK_MFE_APP": {
             "name": "gradebook",
-            "repository": "https://github.com/edx/frontend-app-gradebook",
+            "repository": "https://github.com/ghassanmas/frontend-app-gradebook",
```

Comment:
Let's not forget to change the `ghassanmas` branches to `openedx` before merging. :)

Reply:
We still have this by the way https://github.com/openedx/frontend-app-profile/pull/632